docs/index.rst

```diff
@@ -21,7 +21,9 @@ The following examples are good starting points:
   :ref:`sphx_glr_auto_examples_plot_model_card.py`
 
 In order to better understand the role of each file and their content when
-uploaded to Hugging Face Hub, refer to this :ref:`user guide <hf_hub>`.
+uploaded to Hugging Face Hub, refer to this :ref:`user guide <hf_hub>`. You can
+refer to :ref:`user guide <model_card>` to see how you can leverage model cards
```

Comment:
The link doesn't seem to work for me, see here: https://skops--70.org.readthedocs.build/en/70/index.html. Also, it wouldn't hurt to also list the guide below in the `User Guide / API Reference` section.

Btw. you can catch this when you build the docs:

> checking consistency... /home/vinh/work/forks/skops/docs/model_card.rst: WARNING: document isn't included in any toctree

Reply:
I did `make html` and didn't see it, either have missed it or do I need to run something else?

Reply:
No, I also just did `make html`. It can also be seen in the logs of the rtd build: https://readthedocs.org/api/v2/build/17626522.txt